src/Dropdown/Dropdown.svelte

```diff
@@ -148,15 +148,27 @@
   const dispatchSelect = () => {
     dispatch("select", { selectedId, selectedItem });
   };
-</script>
 
-<svelte:window
-  on:click="{({ target }) => {
+  const pageClickHandler = ({ target }) => {
     if (open && ref && !ref.contains(target)) {
       open = false;
     }
-  }}"
-/>
+  };
+
+  onMount(() => {
+    if (parent) {
+      parent.addEventListener("click", pageClickHandler);
+    }
+  });
```

Comment:
Non-blocking: Why not use the return function in `onMount` instead of `onDestroy`?

```ts
onMount(() => {
  if (parent) {
    parent.addEventListener("click", pageClickHandler);
  }

  return () => {
    // do something
  }
});
```

Reply:
I forgot about this feature! I guess this could simplify things assuming there's no difference between `onDestroy` and this implied `unmount`. Either way I agree it's optional.

Reply:
Never saw the docs about the return function! It's great! I think the `onDestroy` adds more readability but anyway both are good.